.github/workflows/unit-test.yml

```diff
@@ -128,14 +128,14 @@ jobs:
   browser-tests-experimental:
     runs-on: ubuntu-latest
     container:
-      image: circleci/node:14-browsers
+      image: circleci/node:16-browsers
```

Comment:
Starting from `circleci/node:16` the base image uses Debian 10, the stable git version in the registry is v2.20 which met the bar of actions/checkout@v2 (minimum v2.18).